opentreemap/importer/tasks.py

```diff
@@ -144,7 +144,10 @@ def commit_import_event(import_type, import_event_id):
 
     finalize_task = _finalize_commit.si(import_type, import_event_id)
 
-    chord(commit_tasks, finalize_task).delay()
+    async_result = chord(commit_tasks, finalize_task).delay()
```

Comment:
Looking at the [Celery code](https://github.com/celery/celery/blob/master/celery/result.py#L777-L787) suggests that we may need to call `async_result.save()` here (and in other places where we store a GroupResult) in order to later be able to restore the result, though I'm not sure how that works if restoring it via the `AsyncResult` base class `__init__` constructor as opposed the subclass static method `GroupResult.restore`.